bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -35,7 +36,10 @@
 // TODO(GG): for debugging - remove
 // #define DEBUG_SEND_CHECKPOINTS_IN_REVERSE_ORDER (1)
 
+#define getName(var) #var
```

Comment:
```suggestion
#define VARSTR(var) #var
```
or
```suggestion
#define NAME2STR(var) #var
```

Reply:
Looking at the rest of the code, I see that we only use this when we call `toPair(getName(var), var)`. That gets quite repetitive. I suggest turning the whole thing into a macro such that you just write `STRPAIR(var)` or something like that.

Reply:
Yeah, I agree, the use of `toPair(getName(var), var)` is quite repetitive here. Sure, as suggested, I moved to using macro. Thanks